frontend/src/components/users.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import { Heading } from "./headings";
+
+/**
+ * A page to list all of the users that have registered.
+ */
+export const Users = () => {
```

Comment:
`Users` shouldn't be part of this PR, since it's a separate page not affecting the header bar.